epictrack-web/src/components/work/WorkForm/index.tsx

```diff
@@ -224,7 +222,7 @@ export default function WorkForm({ ...props }) {
   useEffect(() => {
     const prefix = getTitlePrefix();
     setTitlePrefix(prefix);
-  }, [work_type_id, project_id]);
+  }, [workTypeId, projectId, projects, workTypes]);

```

Comment:
right of course because getTitlePrefix is depending on projects and workTypes, good catch. I also suggest wrapping the inside of the useEffect by if(projects && workTypes) so we don't do multiple state updates, we just do once, once they are loaded